prow/spyglass/artifacts.go

```diff
@@ -51,7 +51,9 @@ func (s *Spyglass) ListArtifacts(ctx context.Context, src string) ([]string, err
 	}
 
 	artifactNames, err := s.StorageArtifactFetcher.artifacts(ctx, gcsKey)
-	if err != nil {
+	// Don't care errors that are not supposed logged as http errors, for example
+	// context cancelled error due to user cancelled request.
+	if err != nil && !shouldLogHTTPErrors(err) {
```

Comment:
the negation is wrong or not? And could you change the warning to use an error field?

Reply:
You are right, updated :)